src/common/Widget.js

```diff
@@ -135,6 +135,9 @@
         
         cb.apply(null, objs)
     }
+    if (!window.require) {
```

Comment:
Yes - I was trying to suggest that there is no need for ```Widget.prototype.require.``` at all and that you could initialize the root.require up in the UMD declare block before https://github.com/hmsonline/Visualization/blob/GORDON_DEV_FIX/src/common/Widget.js#L6